### DISCUSSION TITLE
gguf Q8 loading slow?

### DISCUSSION THREAD

**Discussion by wzwow** [lllyasviel/stable-diffusion-webui-forge#1596] [General]
it actually failed to load on my system - 32gb, 4070 12gb... what am i missing?

![image](https://github.com/user-attachments/assets/9e7d27fb-2b4a-4791-b8f0-65fadb687d6d)

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1474

**Comment by wzwow** [comment]
double checked i have more than 40gb of virtual memory in my setting. 

every time i load a gguf i get this on cmd and it just stuck there...
![image](https://github.com/user-attachments/assets/87d4f6ef-e3d5-4d40-b5a6-f50d769993d0)

**Reply by wzwow** [reply]
i still cant get gguf to work on forge pls help


![image](https://github.com/user-attachments/assets/2bef94b6-5732-4ff8-a443-2bc02a6d9ee6)

**Comment by wzwow** [comment]
trying Q6.... loading model took 6mins ??? is that normal?

![image](https://github.com/user-attachments/assets/647aebdf-5270-4ca0-8eb4-2388d860839a)
